app/views/pages/resources.html.erb

```diff
@@ -81,8 +81,8 @@
           <li><%= link_to 'Joint Travel Regulation', 'http://www.defensetravel.dod.mil/site/travelreg.cfm', class: "usa-external_link" %></li>
           <li><%= link_to 'Defense Travel Regulation', 'https://www.ustranscom.mil/dtr/', class: "usa-external_link" %></li>
           <li><%= link_to 'Air Force Household Goods Supplement', 'http://static.e-publishing.af.mil/production/1/af_a4_7/publication/afi24-501/afi24-501.pdf', class: "usa-external_link" %></li>
-          <li><%= link_to 'Marine Corps Household Goods Supplement', 'https://www.move.mil/documents/dod/MCO_4600_39_19_AUG_2016.pdf', class: "usa-external_link" %></li>
-          <li><%= link_to 'Navy Household Goods Supplement', 'https://www.move.mil/documents/dod/NAVSUP_PUB_490_Rev5.pdf', class: "usa-external_link" %></li>
+          <li><%= link_to 'Marine Corps Household Goods Supplement', 'https://archive.move.mil/documents/dod/MCO_4600_39_19_AUG_2016.pdf', class: "usa-external_link" %></li>
+          <li><%= link_to 'Navy Household Goods Supplement', 'https://archive.move.mil/documents/dod/NAVSUP_PUB_490_Rev5.pdf', class: "usa-external_link" %></li>
```

Comment:
Pretty sure these need to point to the archive site.

Reply:
Ooh good catch. But we probably shouldn't point people to archive.move.mil... let's download them and throw them in /documents

Reply:
Will do!